acceptance-tests/README.md

```diff
@@ -68,7 +68,7 @@ Note to run the live tests there needs to be a `live.properties` file in the acc
 ```properties
 github.repo=<name of repository to be created
 github.org=<org or user name to create repo in.>
-github.token=<personal access token>  
+github.token=<personal access token (roles: repo, user:email, and optionally delete_repo)>  
```

Comment:
is there any admin needed to setup webhooks? (there is a hyper link that pre-populates) this does look ok to me though, as long as it fits in the visible box

Reply:
hrm no - thanks.. this is good.